docs/source/grid_generation/explanation/database.rst

```diff
@@ -0,0 +1,91 @@
+The Database Architecture of Pylovo
+===================================
+
+.. image:: ../../images/database/database_diagram.png
+    :width: 100%
+    :alt: Database diagram
+
+The database follows a hierarchical structure:
```

Comment:
How is this hierarchical structure related to the categories below: data source tables, version control and configuration, etc...
This is a little confusing here